src/main/java/org/cristalise/kernel/entity/proxy/ItemProxy.java

```diff
@@ -165,7 +209,13 @@ public void initialise(AgentPath agentId, PropertyArrayList itemProps, Composite
         String collString = "";
         if (colls != null) collString = xml.marshall(colls);
 
-        getItem().initialise( agentId.getSystemKey(), propString, wfString, collString);
+        String viewpointString = "";
+        if (viewpoint != null) viewpointString = xml.marshall(viewpoint);
```

Comment:
Here we are checking for null. But some callers put an empty string in these parameters. Maybe StringUtils.isBlank() would be better? Or does null have a different meaning than empty string?